Driverless AI/get-started-with-open-source-custom-recipes-tutorial/get-started-with-open-source-custom-recipes-tutorial.md

```diff
@@ -563,7 +582,7 @@ For this tutorial, we are using **Driverless AI 1.8.7.1,** but we will be workin
 
 ## Next Steps
 
```

Comment:
Please update the tutorial name to the following:

**Build Your Own Custom Recipe Tutorial**